general_superstaq/superstaq_client.py

```diff
@@ -230,28 +232,32 @@ def submit_qubo(self, qubo: qv.QUBO, target: str, repetitions: int = 1000) -> di
         }
         return self.post_request("/qubo", json_dict)
 
-    def find_min_vol_portfolio(self, json_dict: dict) -> dict:
+    def find_min_vol_portfolio(
+        self, json_dict: Dict[str, Union[List[str], int, float, str]]
+    ) -> MinVolJson:
         """Makes a POST request to SuperstaQ API to find a minimum volatility portfolio
         that exceeds a certain specified return."""
         return self.post_request("/minvol", json_dict)
 
-    def find_max_pseudo_sharpe_ratio(self, json_dict: dict) -> dict:
+    def find_max_pseudo_sharpe_ratio(
+        self, json_dict: Dict[str, Union[List[str], float, str, Optional[int]]]
+    ) -> MaxSharpeJson:
         """Makes a POST request to SuperstaQ API to find a max Sharpe ratio portfolio."""
         return self.post_request("/maxsharpe", json_dict)
 
-    def tsp(self, json_dict: dict) -> dict:
+    def tsp(self, json_dict: Dict[str, List[str]]) -> TSPJson:
         """Makes a POST request to SuperstaQ API to find a optimal TSP tour."""
         return self.post_request("/tsp", json_dict)
 
-    def warehouse(self, json_dict: dict) -> dict:
+    def warehouse(self, json_dict: Dict[str, Union[int, List[str], str]]) -> WareHouseJson:
         """Makes a POST request to SuperstaQ API to find optimal warehouse assignment."""
         return self.post_request("/warehouse", json_dict)
 
-    def aqt_upload_configs(self, aqt_configs: Dict[str, str]) -> dict:
+    def aqt_upload_configs(self, aqt_configs: Dict[str, str]) -> Dict[str, str]:
```

Comment:
think this one should just be
```suggestion
    def aqt_upload_configs(self, aqt_configs: Dict[str, str]) -> str:
```
(the docstring in `UserConfig.aqt_upload_configs` is also wrong though)

Reply:
Updated the docstring

Reply:
Committed the suggestion.